lib/pacta.js

```diff
@@ -282,7 +282,8 @@
          * resolve another promise with it. Return that promise as it is equivalent
          * to Promise b.
          */
-        this.map(f).map(function (pb) {
+        var mapped = this.map(f);
```

Comment:
For JSHint/Lint compliance, could you combine this `var` declaration with the `promise` above? Perhaps something like:

```javascript
var promise = new Promise(),
    mappedPromise = this.map(f);
```